src/main/java/reposense/commits/CommitResultAggregator.java

```diff
@@ -155,8 +154,8 @@ private static Date getStartOfDate(Date current, String zoneId) {
      * system's timezone.
      */
     private static Date getSystemStartOfDate(Date current, String zoneId) {
```

Comment:
Can double confirm the usage of this method?

Reply:
My bad for not looking into the detail of these two methods in the previous PR.

Reply:
`getSystemStartOfDate` gets the instant where it says 00:00:00 in the system time zone for the given date. `getStartOfDate` gets the instant where it says 00:00:00 in the given `zoneId` time zone for the given date.

We still need the system time zone because Gson does not allow for a time zone to be specified, so they only recognise the system default time zone.